app/helpers/application_formatters_helper.rb

```diff
@@ -83,6 +83,8 @@ def display_topic(topic, options = {})
   # Format the given datetime
   #
   # @param [DateTime] date The datetime to be formatted
+  # @param [Symbol] format The output format. Use Ruby's defaults or check
```

Comment:
@fonglh actually how about move the DATE_FORMATS declaration to this file ? otherwise, if someone modified  `config/initializers/time_formats.rb` next time, the documentation could mismatch and become harmful:

```
Time::DATE_FORMATS[:date_only_long] = '%B %d, %Y'
Time::DATE_FORMATS[:date_only_short] = '%d %b'

def format_datetime
end
```